pkg/kubelet/dockertools/manager.go

```diff
@@ -1503,9 +1503,9 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 
 	ref, err := kubecontainer.GenerateContainerRef(pod, container)
 	if err != nil {
-		glog.Errorf("Can't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
+		glog.Errorf("can't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
 	}
-	glog.Infof("Generating ref for container %s: %#v", container.Name, ref)
+	glog.V(5).Infof("generated ref for container %s: %#v", container.Name, ref)
```

Comment:
Do we want to print this inside an else {} block? (as in the failure case, we'll see both lines 1506 and 1508 in the logs